src/components/DropzoneArea.js

```diff
@@ -1,15 +1,13 @@
 import Snackbar from '@material-ui/core/Snackbar';
 import Typography from '@material-ui/core/Typography';
 import {withStyles} from '@material-ui/core/styles';
-import CloudUploadIcon from '@material-ui/icons/CloudUpload';
+import {AttachFile, CloudUpload} from '@material-ui/icons/';
 import clsx from 'clsx';
 import PropTypes from 'prop-types';
 import * as React from 'react';
 import {Fragment} from 'react';
 import Dropzone from 'react-dropzone';
-
-import {convertBytesToMbsOrKbs, createFileFromUrl, readFile} from '../helpers';
-
+import {convertBytesToMbsOrKbs, createFileFromUrl, isImage, readFile} from '../helpers';
```

Comment:
Keep the spacing as before

Reply:
I added a function to this, will check the spacing

Reply:
> > Hi there @panz3r I want to fix this now, I will work on fork, should this be as a new commit, or should I modify the old commit, can you help me with the workflow a bit please?
> 
> Hi @max-carroll ,
> 
> I'd try to keep it in the old commit, also (since you'll have to `rebase` a bit 😬 ) reword the second commit to `Add example of how to use the 'getPreviewIcon' prop`.
> The "simplest" way I see is to do the following:
> 
> * create a new commit with the fixes
> * start an interactive `rebase`
> * change the commits order and squash the "fix" commit with the "feature" one
> * reword the "docs" commit
> * complete the `rebase`
> 
> (You'll need to force push afterward but that's ok)

Awesome will do, no problems

Reply:
Oh there is some spacing above it, is that desired?

Reply:
Yeah, it was 😁 but not an actual problem